DigitalOffice/Services/CheckRightsService/Startup.cs

```diff
@@ -30,6 +35,32 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddControllers();
 
+            services.AddMassTransit(configurator =>
+            {
+                configurator.AddConsumer<CheckIfUserHasRightConsumer>();
+
+                configurator.UsingRabbitMq((context, factoryConfigurator) =>
+                {
+                    const string serviceInfoSection = "ServiceInfo";
+
+                    var serviceName = Configuration.GetSection(serviceInfoSection)["Name"];
+                    var serviceId = Configuration.GetSection(serviceInfoSection)["Id"];
+
+                    factoryConfigurator.Host("localhost", "/",hostConfigurator =>
+                    {
+                        hostConfigurator.Username($"{serviceName}_{serviceId}");
+                        hostConfigurator.Password(serviceId);
+                    });
+
+                    factoryConfigurator.ReceiveEndpoint(serviceName, endpointConfigurator =>
+                    {
+                        endpointConfigurator.ConfigureConsumer<CheckIfUserHasRightConsumer>(context);
+                    });
+                });
+            });
```

Comment:
I think, that its configure can move in the new method.

Reply:
fixed